ocaml/xapi/db_gc.ml

```diff
@@ -76,6 +76,53 @@ let gc_VGPU_types ~__context =
 			(String.concat "; " (List.map Ref.string_of (List.map fst garbage)));
 		List.iter (fun (self, _) -> Db.VGPU_type.destroy ~__context ~self) garbage
 
+let gc_PVS_proxies ~__context =
+	(* A proxy connects a VIF, an SR, and a farm. We delete it, only if
+		 all three are invalid. XXX Is it correct to consider cache_SR?  *)
```

Comment:
This comment disagrees with the code. The design states that the proxy should be GC'd if the farm OR the VIF is invalid, so I think the code is right and the comment is wrong.